### PR TITLE
fix: fix null variadic adaptor accumulate row with nullable columns

### PR DESCRIPTION
**Pull request by ariesdevil**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

fix null variadic adaptor accumulate row with nullable columns.

- Closes #issue

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/12715)

### CODE REVIEW

File: src/query/functions/src/aggregates/adaptors/aggregate_null_unary_adaptor.rs [hunk 1]
```diff
@@ -179,7 +179,7 @@ impl<const NULLABLE_RESULT: bool> AggregateFunction for AggregateNullUnaryAdapto
                     return Ok(());
                 }
 
-                if unsafe { v.get_bit_unchecked(row) } {
+                if v.get_bit(row) {
```
Comment (RinChanNOWWW) [inline src/query/functions/src/aggregates/adaptors/aggregate_null_unary_adaptor.rs:182]: Why remove unsafe?

Comment (ariesdevil) [reply src/query/functions/src/aggregates/adaptors/aggregate_null_unary_adaptor.rs:182]: Seems `get_bit` has same behavior as `get_bit_unchecked` without using unsafe.

Comment (RinChanNOWWW) [reply src/query/functions/src/aggregates/adaptors/aggregate_null_unary_adaptor.rs:182]: `get_bit_unchecked`'s performance is better

Comment (ariesdevil) [reply src/query/functions/src/aggregates/adaptors/aggregate_null_unary_adaptor.rs]: fixed.

**Review by RinChanNOWWW — APPROVED** [review]

**Review by b41sh — APPROVED** [review]

**Pull request merged**